project_workflow/__manifest__.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2017 - 2018 Modoolar <info@modoolar.com>
+# License LGPLv3.0 or later (https://www.gnu.org/licenses/lgpl-3.0.en.html).
+
+{
+    "name": "Project Workflow",
+    "summary": "This module provides workflow for project tasks",
+    "category": "Project",
+    "version": "11.0.1.0.0",
+    "license": "LGPL-3",
+    "author": "Modoolar, Odoo Community Association (OCA)",
+    "website": "https://www.modoolar.com/",
```

Comment:
@pnajman-modoolar IMHO Change URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10)